apps/classroomio-com/src/routes/teachers/+page.svelte

```diff
@@ -0,0 +1,216 @@
+      showAnswer: false
+    }
+  ];
+
+  function toggleAnswer(index) {
+    faqs[index].showAnswer = !faqs[index].showAnswer;
+  }
+</script>
+
+<svelte:head>
+  <title>Teacher | ClassroomIO</title>
+</svelte:head>
+
+<div class="mt-[10%] md:mt-16">
+  <header class="root bg-[#F5F8FE] px-[10%] border-4 pt-14 w-full flex flex-wrap text-left">
+    <div class="lg:max-w-2/4 pt-[6%] pb-8">
+      <h1 class="text-5xl font-bold">Teach with us</h1>
+      <p class="my-5 lg:w-[77%] lg:text-left text-center">
+        With ClassroomIO, you can make side income while teaching the subject you love
+      </p>
+      <button type="button" class="w-full lg:w-[30%] py-3 rounded-md text-white bg-[#1D4ED8]"
+        >Register</button
+      >
+    </div>
+
+    <div class="lg:max-w-[40%] z-20">
+      <img src="/teacher-hero.png" alt="" class="w-full z-20" />
+    </div>
+  </header>
+
+  <div class="mt-10 mx-auto">
+    <div class="px-[10%]">
+      <h1 class="text-2xl text-center font-bold flex gap-3 justify-center items-center">
+        Why you should start with us <img src="/star.svg" alt="" />
+      </h1>
+
+      <div class="flex flex-wrap justify-center lg:justify-between mt-10">
+        {#each whyStart as why}
+          <figure
+            class="flex gap-3 flex-col justify-center items-center px-2 w-full h-full max-w-[250px] max-h-[380px] text-center"
+          >
+            <img src={why.src} alt="" class="w-[80px] h-[80px]" />
+            <figcaption>
+              <h1 class="font-bold text-sm">{why.header}</h1>
+              <p class="text-xs mt-1">
+                {why.subText}
+              </p>
+            </figcaption>
+          </figure>
+        {/each}
+      </div>
+    </div>
+
+    <div class="px-[10%] mt-10 py-10">
+      <h1 class="text-2xl text-center font-bold flex gap-3 justify-center items-center">
+        How to get started <img src="/thumbs-up.svg" alt="" />
+      </h1>
+
+      <div class="flex flex-wrap justify-between gap-y-10 mx-auto mt-5">
+        {#each steps as step, i}
+          <div class="lg:max-w-[40%] px-10 py-5">
+            <h1 class="font-bold">
+              <span class="px-2 py-1 mr-3 bg-[#F7F7F7] rounded-md">{i + 1}</span>
+              {step.title}
+            </h1>
+            <p class="text-xs text-gray-600 mt-5">
+              {step.subText}
+            </p>
+          </div>
+        {/each}
+      </div>
+      <div class="flex justify-end px-[5%] mt-3">
+        <button
+          type="button"
+          class="w-full lg:max-w-[20%] py-3 text-xs text-white rounded-md bg-[#1D4ED8]"
+          >Click here to register</button
+        >
+      </div>
+    </div>
+
+    <div class="px-[12%] bg-[#F5F8FE] my-10 py-12 h-[75%] lg:h-[80%]">
+      <h1 class="text-2xl text-center font-bold flex gap-3 justify-center items-center">
+        Word from our fufilled facilitators
+      </h1>
+
+      <img src="/facilitator-1.png" class="lg:w-2/4 mt-10 mx-auto" alt="" />
```

Comment:
This should not be an image.

It should be html/css cause the text needs to be copied and it can be crawled by google.